en/micro-integrator/docs/references/connectors/kafka-connector/kafka-connector-configuration.md

```diff
@@ -0,0 +1,45 @@
+# Kafka Connector Configuration
+
+[[Setting up the environment]](#Setting-up-the-environment) 
+
+The Kafka connector allows you to access the Kafka Producer API through WSO2 EI and acts as a message producer that facilitates message publishing. The Kafka connector sends messages to the Kafka brokers.
+
+The Kafka is a distributed publish-subscribe messaging system that maintains feeds of messages in topics. Producers write data to topics and consumers read from topics. For more information on Apache Kafka, see [Apache Kafka documentation](http://kafka.apache.org/documentation.html).
+It provides the functionality of a publish-subscribe messaging system with a unique design.The Kafka mainly operates based on a topic model. A topic is a category or feed name to which records get published. Topics in Kafka are always multi-subscriber.
+
+## Setting up the environment
+
+To use the Kafka connector, download and install [Apache Kafka](http://kafka.apache.org/downloads.html).
```

Comment:
Lets add something that says you also need WSO2 EI and we refer to that location as <PRODUCT_HOME>.

Reply:
Added new line for this.